python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -185,15 +185,35 @@ def build_run_status_sensor_context(
     )
 
 
+class JobAddress(
+    NamedTuple(
+        "_JobAddress",
+        [("repository_location", str), ("repository", str), ("job_name", str)],
```

Comment:
should this be location_name, repository_name, job_name?

Reply:
yeah, one of the questions i have is on the naming and terminology. I'm still a bit unclear on the specific differences between `code_location` `repository_location` or just `location`

Reply:
they're all the same.

Reply:
I think we're trying to move away from `repository_location`